guides/8-perpetual-exchange/2-how-it-works.md

```diff
@@ -0,0 +1,346 @@
+* `Total Tokens in Pool`: 1,000 SOL
+* `Hourly Borrow Rate`:  0.01% (0.0001 in decimal format / 1 BPS)
+* `Position Size`: $10,000
+
+Calculation:
+
+```
+Hourly Borrow Fee = (90 / 1000) * 0.0001 * 10000 = 0.09
+```
+
+This means your position will accrue a borrow fee of $0.09 every hour it remains open.
+
+:::info
+Borrow fees are continuously accrued and deducted from your collateral. This ongoing deduction has two important consequences:
+
+1. Your effective leverage increases over time as your collateral decreases.
+2. Your liquidation price moves closer to the current market price.
+
+It's crucial to regularly monitor your borrow fees and liquidation price. Failure to do so may result in unexpected liquidation, especially during periods of high market volatility or extended position duration.
+:::
+
+### Transaction & Priority Fee
+
+Traders will have to pay SOL for submitting transactions onto the Solana chain. Traders also pay priority fees or Jito bundle tips (or both) depending on their settings.
+
+At the same time, a minor SOL amount will be used for rent to create an escrow account ([PDA](https://solanacookbook.com/core-concepts/pdas.html#facts)). The SOL rent will be returned to you once you close your position.
+
+## Example Trade
+
+With all these concepts covered, let's go through a worked example.
+
+Suppose a trader wants to open a 2x long SOL position at a position size of $1000 USD by depositing $500 USD worth of SOL as a collateral and borrowing $500 USD worth of SOL from the pool.
+
+| Initial Position Value | $1000 |
+| --- | ----- |
+| Initial Deposit | $500 |
+| Borrowed Amount | $500 |
+| Leverage | 2x |
+| Initial SOL Price | $100 |
+| Utilization Rate | 50% |
+| Borrow Rate | 0.008% per hour |
+| Position Opening Fee | `0.06% * $1000 = $0.6` |
+
+The trader keeps this position open for 2 days, and the price of SOL appreciates by 10%.
+
+| Final Position Value | $1100 |
+| --- | ----- |
+| Final SOL Price | $110 |
+| Holding Period | 2 days (48 hours) |
+| Position Closing Fee | `0.06% * $1100 = $0.66` |
+
+The borrow fee accumulated throughout this period can be calculated as:
+
+- `Hourly Borrow Fee = Tokens Borrowed/Tokens in the Pool * Borrow Rate * Position Size`
+- `Total Borrow Fee = 50% * 0.008% * 1000 * 48 = $1.92 USD`
+
+
+The trader's final profit can be calculated as:
+
+- `Final Profit = Final Position Value - Initial Position Value - Borrow Fee - Opening Fee - Closing Fee`
+- `$1100 - $1000 - $1.92 - $0.6 - $0.66 = $96.82`
+
+The trader gets a final profit of **$96.82 USD** after this trade.
+
+
+## Oracle
+
+Jupiter Perps uses a new Oracle Network, co-designed by Jupiter and another key ecosystem player (to be announced at Breakpoint), and audited by Offside Labs.
+
+This oracle, known as Signal, is used as the mark price for opening and closing trades, calculating margin requirements, and triggering SL/TP orders and liquidations on the Jupiter platform.
```

Comment:
Chaos Labs has now changed the name to `Edge`: https://www.binance.com/en/square/post/2024-09-13-chaos-labs-launches-decentralized-oracle-protocol-edge-for-solana-ecosystem-13468079116865

Reply:
I think the latest name is Dove - will confirm and push the latest change